news/2022-05-25-new-featured-artist-origami-angel.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "New Featured Artist: Origami Angel"
+date: 2022-05-25 13:00:00 +0000
+---
+
+**Origami Angel** joins the fold as latest our Featured Artist!
+
+![](https://assets.ppy.sh/artists/269/header.jpg)
+
+Showcasing the cutting-edge state of pop-punk and emo music, **Origami Angel** are an absurdly talented duo who's renowned for having tons of different tempos and styles while always maintaining upbeat, exciting energy. In the last year, the osu! community has shown a massive upsurge of interest in the band, so it should be no surprise that they're here today!
+
+We've picked up **17** tracks from the band, including a bunch from their newest *GAMIGANG* album and the entirety of *Somewhere City*! Check out the full list from [**Origami Angel**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/269).
+
+Ever wondered how many people recognize osu! out in the wild? The answer is *a lot*, especially if you're at a tournament full of dedicated gamers. Watch some Super Smash Bros. players attempt [Origami Angel - The Title Track](https://osu.ppy.sh/beatmapsets/1396156) on a makeshift setup, then check out a bunch more maps to play for yourself:
+
+<div align="center">
+    <video width="85%" controls>
+        <source src="https://assets.ppy.sh/artists/269/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### The Title Track
+
+Check out [the map from the video above](https://osu.ppy.sh/beatmapsets/1396156) hosted by [Mimari](https://osu.ppy.sh/users/14339830) or [this healthier alternative](https://osu.ppy.sh/beatmapsets/1503591) hosted by [Jemzuu](https://osu.ppy.sh/users/7890134)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/269/Somewhere%20City%2FOrigami%20Angel%20-%20The%20Title%20Track.mp3" type="audio/mpeg">
+</audio>
+
+### 24 Hr Drive-Thru
+
+If you're looking for something easier, try [this chill mapset](https://osu.ppy.sh/beatmapsets/1724847) hosted by [SMOEKLIND](https://osu.ppy.sh/users/9327302)!
```

Comment:
```suggestion
If you're looking for something easier, try [this chill mapset](https://osu.ppy.sh/beatmapsets/1724847) hosted by [SMOKELIND](https://osu.ppy.sh/users/9327302)!
```

Reply:
this is actually smokelind's dutch cousin